src/test/java/org/folio/edge/sip2/handlers/EndPatronSessionHandlerTests.java

```diff
@@ -1,30 +1,38 @@
 package org.folio.edge.sip2.handlers;
 
+import static java.lang.Boolean.TRUE;
 import static org.folio.edge.sip2.parser.Command.END_SESSION_RESPONSE;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
 
 import freemarker.template.Template;
+import io.vertx.core.Future;
 import io.vertx.core.Vertx;
 import io.vertx.junit5.VertxExtension;
 import io.vertx.junit5.VertxTestContext;
-
 import java.time.Clock;
 import java.time.OffsetDateTime;
 import java.time.format.DateTimeFormatter;
 
 import org.folio.edge.sip2.api.support.TestUtils;
 import org.folio.edge.sip2.domain.messages.requests.EndPatronSession;
+import org.folio.edge.sip2.domain.messages.responses.EndSessionResponse;
 import org.folio.edge.sip2.handlers.freemarker.FreemarkerRepository;
+import org.folio.edge.sip2.repositories.PatronRepository;
 import org.folio.edge.sip2.session.SessionData;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-@ExtendWith({VertxExtension.class})
+@ExtendWith({VertxExtension.class, MockitoExtension.class})
 public class EndPatronSessionHandlerTests {
   @Test
-  public void canSuccessfullyGetEndPatronSessionResponse(Vertx vertx,
-                                                         VertxTestContext testContext) {
+  public void canSuccessfullyGetEndPatronSessionResponse(
```

Comment:
Should we have a test case for when it cannot successfully end the patron session, because now we have to check the user and the user may or may not be found?  The test case PatronRepositoryTets.cannotPatronEndSessionRequirePasswordWithBadPassword() does this verification, but I was wondering how the handler would react to it.

Reply:
Sure, I'll add one. The only difference should be `endSession` being set to `FALSE`, but it'd be nice to prove that.